exercises/beer-song/.meta/generator/beer_song_case.rb

```diff
@@ -17,7 +17,7 @@ def beer_song_arguments
     if property == 'verse'
       number
     else
-      "%s, %s" % [self["beginning"], self["end"]]
+      "%s, %s" % [self.beginning, self.end]
```

Comment:
These names come from the canonical data so we're constrained by what it provides.